sensor-helm-chart/values.yaml

```diff
@@ -8,7 +8,7 @@ image:
   repository: registry.digitalocean.com/odetect/odetect-prod
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: 'e5f33b3'
```

Comment:
Check why this changed.

Reply:
Fixed.

I'd forgotten to push the tag update commit to devenv that we made during the deployment yesterday. So I did that and then re-rebased.